extensions/wikia/AdEngine/js/recovery/helper.js

```diff
@@ -41,11 +44,11 @@ define('ext.wikia.adEngine.recovery.helper', [
 	}
 
 	function isRecoveryEnabled() {
-		return !!(context.opts.sourcePoint && SourcePointTag);
+		return !!(context.opts.sourcePointRecovery && SourcePointTag);
```

Comment:
Good point! Not whole `AdContext` can be cached on Varnishes :) Let's leave it then.